nmdc_runtime/site/translation/submission_portal_translator.py

```diff
@@ -557,7 +557,7 @@ def _translate_biosample(
         biosample_key = sample_data[0].get(BIOSAMPLE_UNIQUE_KEY_SLOT, "").strip()
         slots = {
             "id": nmdc_biosample_id,
-            "part_of": nmdc_study_id,
+            "associated_studies": nmdc_study_id,
```

Comment:
Since [associated_studies](https://microbiomedata.github.io/berkeley-schema-fy24/associated_studies/) is a _multivalued_ slot, should we have this line of code be?

```python
"associated_studies": [nmdc_study_id]
```

Reply:
yes, this should be an array.

Reply:
`part_of` back in ye olde schema was also multivalued. LinkML-generated `@dataclass`es will automatically cast scalars into lists with a single element when needed. So that's why it technically works, but yeah it's still better to be explicit and not rely on the implicit conversion. I will commit an update.